include-what-you-use.1

```diff
@@ -63,7 +63,7 @@ default.
 .RE
 .TP
 .B \-\-cxx17ns
-Suggest the more concise syntax for nested namespaces introduced in C++17.
+Use C++17 nested namespaces when suggesting additions of forward declarations.
```

Comment:
Ultra-nit: Here "Use" starts with capital U whereas in the other file it's small u.

Reply:
That's intentional, to vibe with the surrounding code